src/classes/InstallScript.cls

```diff
@@ -44,4 +68,14 @@ public without sharing class InstallScript implements InstallHandler {
         packageSettings.NamespacePrefix__c = VOL_SharedCode.StrTokenNSPrefix('');
         insert packageSettings;
     }
+
+    @TestVisible
+    private void sendUsageMetrics() {
+        try {
```

Comment:
@lmeerkatz - I added it in the method itself. Does that work or would you prefer it up top?

Reply:
That works. Thank you!